troubleshooting/known_issues/enable_iscsid_multipathd.md

```diff
@@ -0,0 +1,214 @@
+```
+
+**Note:** use above $USERNAME and $PSWD for VCS login
+
+Step-4: Apply fix against import_branch found in Step-2
+
+**Example**
+
+```bash
+ncn-m001:/home/ # cd csm-config-management
+ncn-m001:/home/csm-config-management # git checkout cray/csm/1.27.2
+ncn-m001:/home/csm-config-management # git checkout -b CASMTRIAGE-7509
+```
+**Note: ** `cray/csm/1.27.2` is a target branch and `CASMTRIAGE-7509` is a new branch
+
+Add new role to enable `iscsid` and `multipathd` service:
+
+```bash
+ncn-m001:~/csm-config-management # cat roles/csm.enable_iscsid_multipathd/tasks/main.yml
+---
+- name: Ensure iscsid service is started
+  ansible.builtin.systemd:
+    name: iscsid
+    state: started
+    enabled: true
+
+- name: Ensure multipathd service is started
+  ansible.builtin.systemd:
+        name: multipathd
+        state: started
+        enabled: true
+ncn-m001:~/csm-config-management #
+```
+
+Apply below changes to “csm-config-management/csm_packages.yml” to Application-nodes only play and Compute-nodes only play under “csm_services” in order to enable `iscsid` and `multipathd` services.
+
+```bash
+diff --git a/csm_packages.yml b/csm_packages.yml
+index e3366f8..b223aec 100755
+--- a/csm_packages.yml
++++ b/csm_packages.yml
+@@ -137,6 +137,9 @@
+       vars:
+         packages: "{{application_csm_sles_packages }}"
+       when: ansible_distribution_file_variety == "SUSE"
++    # Enable iscsid and multipathd service
++    - role: csm.enable_iscsid_multipathd
++
+   tasks:
+     - name: Enable smart service
+       systemd:
+@@ -148,3 +151,12 @@
+         name: cray-node-exporter
+         state: started
+         enabled: true
++
++# Compute-nodes only play
++- hosts: Compute:!cfs_image
++  gather_facts: no
++  any_errors_fatal: true
++  remote_user: root
++  roles:
++    # Enable iscsid and multipathd service
++    - role: csm.enable_iscsid_multipathd
+```
+
+Step-5: Commit above changes and push the changes to VCS
+
+**Example**
+
+```bash
+ncn-m001:/home/csm-config-management # git add csm_packages.yml
+ncn-m001:/home/csm-config-management # git commit -m "fix for CASMTRIAGE-7509"
+ncn-m001:/home/csm-config-management # git push --set-upstream origin CASMTRIAGE-7509
+
+ncn-m001:/home/csm-config-management # COMMIT=`git log | head -n 1 | awk '{print $2}'`
+ncn-m001:/home/csm-config-management # echo $COMMIT
+bf214b8a9867531a38f8ca28b6ffae1fe56724ce
+```
+
+Step-6: Create new CFS configuration with above change to be applied
+
+**Example**
+
+```bash
+ncn-m001:/home/csm-config-management # SESSIONTEMPLATE=compute-25.1.0-alpha2.x86_64-csm-160-rc4
+ncn-m001:/home/csm-config-management # CFS_CONFIG=`cray bos sessiontemplates describe $SESSIONTEMPLATE --format json | jq -r .cfs.configuration`
+ncn-m001:/home/csm-config-management # cray cfs configurations describe $CFS_CONFIG --format json | jq '. | del(.lastUpdated) | del(.name)'  > $CFS_CONFIG
+```
+
+Step-7: Update commit id ($COMMIT from step 5 in $CFS_CONFIG for “csm_packages.yml”)
```

Comment:
Put both `$COMMIT` and `$CFS_CONFIG` in back-ticks, and change to back-tics for `csm_packages.yml`.